docs/mycoind/events.rst

```diff
@@ -6,38 +6,13 @@ As we have a websocket connection with the tendermint
 server, we can not only query accounts and post transactions
 but also react to events. This is very nice for modern
 interfaces, but also less thoroughly tested (both in tendermint
-and in weave-js) and there may be some bumps here, especially
+and in iov-core) and there may be some bumps here, especially
 with new versions of tendermint.
 
 Listening for Headers
 ---------------------
 
-.. code:: javascript
-
-    // subscribe to all new headers, and store in an array
-    let headers = []
-    client.subscribeHeaders(x => headers.push(x))
-    // query the length a few times to see it grow
-    headers.length
-    headers.length
-    // unsubscribe and check that no new headers are added
-    client.unsubscribe()
-    headers.length
-    headers.length
-    // look at one header to see what info is available
-    let mine = headers[1]
-    pprint(mine)
-    // you get similar info by querying
-    let height = mine.block.header.height
-    let head = await client.header(height)
-    let block = await client.block(height)
-    pprint(head)
-    pprint(block)
-    pprint(mine)
-    // one has block, other block_meta....
-    JSON.stringify(block.block) === JSON.stringify(mine.block)
-    JSON.stringify(head) === JSON.stringify(block.block_meta)
-
+** TODO** add iov-core example when https://github.com/iov-one/iov-core/pull/317 done.
```

Comment:
This should be unblocked now.
There are some examples here: https://github.com/iov-one/iov-core/tree/master/packages/iov-core#reactive-clients

If this is missing something, let me know and I will add to both docs.